portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/oauth2/OAuth2.testcase

```diff
@@ -603,6 +603,94 @@ definition {
 			resourcePanels = "Portal Services,Sample QA app");
 	}
 
+	@description = "This is a use case for LRQA-57134."
```

Comment:
Can we cite the story ticket this test is based off of? Or we can not specify a description at all. We can track down what LRQA added this test in the Github history.

Reply:
I change the description of the LPS story ticket number, instead of the LRQA item.